Examples Folder

antonplietzsch opened the pull request:
1) I added an examples folder to the PowerDynamics repo as discussed in #119.
2) As a first example i added the IEEE 14-bus system with 4th order generators.
3) I also added tests for this example to check that the fault simulations run through successfully.

SabineAuer commented on test/examples/ieee14bus.jl:
```diff
@@ -0,0 +1,69 @@
+
+    solution3 = simulate(fault3, pg_build, operationpoint, timespan)
+    @test solution3.dqsol.retcode == :Success
+end
```
@antonplietzsch @luap-pik  I think it would also be nice to check if the solution is reproduced

SabineAuer commented on test/examples/ieee14bus.jl:
```diff
@@ -0,0 +1,69 @@
+using PowerDynamics
+using OrderedCollections: OrderedDict
+
+buses=OrderedDict(
```
@antonplietzsch It would be handy to include the code from examples/ieee14bus/buildexample.jl here simply instead of defining the IEEE14 bus system two times in the repo?

antonplietzsch replied:
Yes, good point. I'll do that.

luap-pik commented on examples/ieee14bus/buildexample.jl:
```diff
@@ -0,0 +1,46 @@
+    "bus1"=> FourthOrderEq(T_d_dash=7.4, D=2, X_d=0.8979, X_q=0.646, Ω=50, X_d_dash=0.2995, T_q_dash=0.1, X_q_dash=0.646, P=2.32, H=5.148, E_f=1),
+    "bus2"=> SlackAlgebraic(U=1),
+    "bus3"=> FourthOrderEq(T_d_dash=6.1, D=2, X_d=1.05, X_q=0.98, Ω=50, X_d_dash=0.185, T_q_dash=0.4, X_q_dash=0.36, P=-0.942, H=6.54, E_f= 1),
+    "bus4"=> VoltageDependentLoad(P=-0.478, Q=-0.0, U=1.0, A=0.0, B=0.0),
```
You are using `VoltageDependentLoad`'s here but the factors are all `0`. Is that for a reason?

antonplietzsch replied:
Not really. I thought it might be easier for users to play around with different load models by just changing parameters instead of replacing node models. But you're right, it is a bit confusing right now. I think I will either go back to the `PQAlgebraic` or change the parameters in the `VoltageDependentLoad` to make it actually voltage dependent.

SabineAuer commented:
@antonplietzsch : was there anything else you wanted to add here with Paul? @luap-pik?

antonplietzsch commented:
I changed the following things in the last commit:

1. Loads are now modeled as `PQAlgebraic`
2. I included the _buildexample.jl_ in the test, such that the test system is only defined once in the repo.
3. I simplified the plotting script a bit.

SabineAuer reviewed (approved):
Review done together with  @antonplietzsch  during zoom call
